### ISSUE TITLE
Add HTML language

### ISSUE THREAD

**Issue by dkniffin** [tmont/sunlight#17]
If I put ````html in my jsdoc tutorials, it doesn't highlight the syntax. I think this could easily be resolved by making that render as xml.

Along the same lines, js should be recognized as the same as javascript

**Comment by tmont** [comment]
Yeah, it doesn't support aliases currently, but that would be a good feature for sure. I'll leave this issue open.
